File: app/src/main/java/com/inquallity/daggersandbox/BootstrapActivity.java
package com.inquallity.daggersandbox;

import android.content.Intent;
import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.inquallity.daggersandbox.main.MainActivity;

public class BootstrapActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        startActivity(new Intent(this, MainActivity.class));
    }
}

File: app/src/main/java/com/inquallity/daggersandbox/BootstrapActivity.kt
package com.inquallity.daggersandbox

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.inquallity.daggersandbox.main.MainActivity

class BootstrapActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        startActivity(Intent(this, MainActivity::class.java))
    }
}